FileUpload/src/FileUpload.jsx

```diff
@@ -399,13 +405,13 @@ export default class FileUpload extends Component {
                 {state.selectedFile &&
                     <div className="dnn-file-upload-file-name"><span>{state.selectedFile.value}</span></div>}
             </div>
-            {state.uploading && <UploadBar 
+            {state.uploading && <UploadBar
                 uploadCompleteText = {props.uploadCompleteText}
                 uploadingText = {props.uploadingText}
                 uploadDefaultText = {props.uploadDefaultText}
```

Comment:
The prop does not make sense anymore. Please remove from `UploadBar`. Unless it can be replaced with something better?